.github/workflows/scripts/createTestEnvironments.sh

```diff
@@ -37,7 +37,8 @@ createRegularEnv() {
   --data-raw "{
       \"name\": \"$CSAE_ENV_NAME\",
       \"region\": \"us-central\",
-      \"password\": \"$CSAE_ENV_PASSWORD\"
+      \"password\": \"$CSAE_ENV_PASSWORD\",
+      \"startupScript\": \"#!/bin/bash\n\nfunction withRetry {\n  local RETRIES=\$1; shift 1\n  local SLEEP=\$1; shift 1\n  for i in \$(seq 1 \$RETRIES)\n  do\n    echo \\\"Attempt \$i: Running command \$@\\\"\n    \$@ && s=0 && break || s=\$? && sleep \$SLEEP\n  done\n  return \$s\n}\n\nfunction ensureVantageIsUp {\n  pdestate -a\n  pdestate -a | grep \\\"DBS state is [45]\\\"\n}\n\ndbscontrol << EOF\nm g 53=P\nW\nEOF\ntpareset -y changing dbscontrol\nwithRetry 40 2 ensureVantageIsUp\"
```

Comment:
like we discussed in the email, changes to `g 53` do not require a restart. You can simplify this startup script to something like this (have not tested, sharing this as an example):

```
\"startupScript\": \"#!/bin/bash\n\ndbscontrol << EOF\nm g 53=P\nW\nEOF\n"
```